pandas/tests/groupby/test_function.py

```diff
@@ -1234,3 +1234,16 @@ def test_groupby_sum_timedelta_with_nat():
     res = gb["b"].sum(min_count=2)
     expected = Series([td3, pd.NaT], dtype="m8[ns]", name="b", index=expected.index)
     tm.assert_series_equal(res, expected)
+
+
+def test_groupby_empty_dataset():
```

Comment:
Can you add parameters e.g. percentiles, include, exclude.

Reply:
Sure. But, I do not add 'exclude' parameter in this particular test case. The reason is that `DataFrameGroupBy` data and `SeriesGroupBy` data are empty.

Reply:
But the columns still have dtypes which can be excluded, and we should ensure consistency between empty/non-empty. For example, column "a" is excluded here:

```
df = pd.DataFrame(columns=["a", "b", "c"])
df["a"] = df["a"].astype(int)
df["b"] = df["b"].astype(float)
print(df.describe(exclude=int))

          b    c
count   0.0    0
unique  NaN    0
top     NaN  NaN
freq    NaN  NaN
mean    NaN  NaN
std     NaN  NaN
min     NaN  NaN
25%     NaN  NaN
50%     NaN  NaN
75%     NaN  NaN
max     NaN  NaN
```

Reply:
Great. Have updated